cocos/2d/CCFastTMXTiledMap.cpp

```diff
@@ -217,20 +215,16 @@ TMXLayer * TMXTiledMap::getLayer(const std::string& layerName) const
 
 TMXObjectGroup * TMXTiledMap::getObjectGroup(const std::string& groupName) const
 {
-    CCASSERT(!groupName.empty(), "Invalid group name!");
-    if (groupName.empty()) {
-        return nullptr;
-    }
-
-    if (!_objectGroups.empty())
+    CCASSERT(groupName.size() > 0, "Invalid group name!");
```

Comment:
Generally, we use empty to check whether a container is empty rather than size > 0.
And if i remember correctly, the `if (groupName.empty()) { return nullptr;}` was added by @WenhaiLin for not causing crash in release mode. Therefore, why to remove this logic?